frameworks/hdfs/src/main/dist/ssl-client.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+    <property>
+        <name>ssl.client.truststore.password</name>
+        <value>changeit</value>
```

Comment:
These are well-known passwords. And making them configurable won't make sense until we start generating keystores and truststores.